### ISSUE TITLE
Obsoletion candidate: Abnormality of glycoprotein metabolism

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4291]
We are using this now as a grouping term. The two leaf terms underneath it are enzyme activities that belong under here: https://hpo.jax.org/app/browse/term/HP:0012379. I think we can obsolete/merge this term as it is not a useful group and is likely to confuse users. @drseb

**Comment by drseb** [comment]
You don't like the idea of moving "Abnormality of glycoprotein metabolism" under "Abnormal enzyme/coenzyme activity"? If not, then go ahead and obsolete?

**Comment by pnrobinson** [comment]
Abnormality of glycoprotein metabolism is not a subclass of abn enzyme activity. I thnk when we originally made this term we were expecting to have many more terms like it. But with the decision to not use "homeostasis..." etc term but instead use "concentration..." etc terms, we need to restructure a little bit! OK, I will merge.

**Comment by matentzn** [comment]
The term is missing AFAIK the 'obsolete' in the label. Just for consistency maybe add it back in.

**Comment by matentzn** [comment]
'Abnormality of glycoprotein metabolism' -> 'obsolete Abnormality of glycoprotein metabolism'

**Comment by drseb** [comment]
yes. what is the status of the obsoletion plugin? is this not working for the editors?

**Comment by drseb** [comment]
fixed
